prodactivity-frontend/components/pages/SplashPage.tsx

```diff
@@ -2,16 +2,13 @@ import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text } from 'react-native';
 
 export default function SplashPage() {
```

Comment:
Another minor thing, but we should use arrow functions over the 'function' keyword. The linter can probably switch a lot of this over when it gets setup.